src/TestInfrastructure/NoOpThreadHandler.cs

```diff
@@ -46,10 +46,14 @@ public Task RunOnUIThread(Action op)
 
         public void RunOnUIThreadSync(Action op) => op();
 
+        public void RunOnUIThreadSync2(Action op) => op();
+
         public Task<T> RunOnBackgroundThread<T>(Func<Task<T>> asyncMethod) => asyncMethod();
 
         public IAwaitableWrapper SwitchToBackgroundThread() => new NoOpAwaitable();
 
+        public Task SwitchToMainThreadAsync() => Task.CompletedTask;
```

Comment:
This was on a separate commit than the original SwitchToMainThreadAsync add-in was so it was simpler for me to just add in this line. (I made sure it is in the same place.)